src/main/java/net/glowstone/entity/GlowAnimal.java

```diff
@@ -44,6 +44,30 @@ protected int getAmbientDelay() {
         return 120;
     }
 
+    /**
+     * Determines whether this entity can eat an item, if so, applies the effects of eating it.
+     *
+     * @param player the player feeding the entity, for statistical purposes
+     * @param type an item that may be food
+     * @return true if the item should be consumed; false otherwise
+     */
+    protected boolean tryFeed(Material type, GlowPlayer player) {
+        if (getBreedingFoods().contains(type)) {
+            if (canBreed() && (getInLove() <= 0)) {
```

Comment:
The parentheses for the second condition are unnecessary.

Reply:
Removed, although I would've thought they'd improve readability (since `getInLove()` _looks_ like the name of a method that would return boolean).